ludwig/constants.py

```diff
@@ -183,6 +183,14 @@
 CONTINUE_PROMPT = "Do you want to continue? "
 
 DEFAULT_AUDIO_TENSOR_LENGTH = 70000
+AUDIO_FEATURE_KEYS = [
```

Comment:
idk if this would be too complicated, but an alternate pattern you could use is to make an AudioFeatureKeys schema that is additionally derived by your audio feature schema. Then instead of having this list of keys, it would be a matter of checking if a feature is in that list of attributes. But this is clean, just throwing out an idea if it would be easier to maintain :P

Reply:
So I think this would work pretty well and would have the added benefit of having things more schema centralized. On the counter argument, it would probably be like 5x the amount of code since I would have to build out this sub schema whereas here it's pretty small and I don't think there are any plans to get rid of constants. My final vote would be to just keep this implementation, but if you/Justin feel that the other way offers more benefits, I am down to change it.

Reply:
totally up to you, ya if this list isn't going to change then current approach is much simpler